docs/api/fleet/agent_api_checkin.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[fleet-agent-checkin]]
+=== Fleet agent checkin API
+++++
+<titleabbrev>Agent checkin</titleabbrev>
+++++
+
+This API is used by a fleet agent to report is current state.
+
+[[fleet-agent-checkin-request]]
+==== Request
+
+`POST /api/fleet/agents/{agentId}/checkin` 
+
+==== Headers
+
+`kbn-fleet-access-token`::
+  (Required, string) A fleet agent access token.
+
+[[fleet-agent-checkin-request-body]]
+==== Request body
+
+`events`::
+  (Required, array) An array of events with the properties `type`, `subtype`, `message`, `timestamp`, `payload`, and `data`.
+
+`local_metadata`::
+  (Optional, object) an object that contains the local metadata for an agent. Metadata are a dictonary of strings (example: `{ "os": "macos" }`).
```

Comment:
```suggestion
  (Optional, object) An object that contains the local metadata for an agent. The metadata is a dictionary of strings (example: `{ "os": "macos" }`).
```

Reply:
I know that grammatically it makes sense to make metadata plural, but "Metadata are a dictionary" sounds odd to my ear because dictionary is single.